test-app/.template-lintrc.js

```diff
@@ -36,13 +36,13 @@ module.exports = {
       },
     },
     {
-      files: ['**/tests/dummy/**/*.hbs', '**/tests/**/*.{js,ts}'],
+      files: ['**/*.hbs', '**/tests/**/*.{js,ts}'],
```

Comment:
Running `yarn lint` with this file content works, I believe other rules/overrides are not needed in `test-app` directory (they are only needed for files in `./ember-slugify/src/*`)
```js
'use strict'

module.exports = {
  extends: ['recommended', 'stylistic'],

  overrides: [
    {
      files: ['**/tests/**/*.{js,ts}'],
      rules: {
        'eol-last': 'off',
      },
    },
  ],
}
```

Reply:
Hey, I think I am going to merge the PR as it is once green because it really starts to hit hard on my mental health and I want to unblock the dependencies update. Let's have a better look at the lint config afterward :)